test/archiver.js

```diff
@@ -6,7 +6,7 @@ const zlib = require('zlib');
 const stream = require('stream');
 const assert = require('assert');
 const sinon = require('sinon');
-const sandbox = sinon.sandbox.create();
+const sandbox = sinon.createSandbox();
```

Comment:
creating sandbox in the latest version of sinon is ever so slightly different 
https://sinonjs.org/releases/v15/sandbox/